src/signals/IncidentMap/components/IncidentMap/utils.test.ts

```diff
@@ -0,0 +1,31 @@
+import type { Map } from 'leaflet'
```

Comment:
It's missing 

// SPDX-License-Identifier: MPL-2.0
// Copyright (C) 2022 Gemeente Amsterdam